Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -80,10 +80,11 @@ BreakPoints::TBreakPointsStr BreakPoints::GetStrings() const
 
 void BreakPoints::AddFromStrings(const TBreakPointsStr& bp_strings)
 {
-  for (const std::string& bp_string : bp_strings)
+  for (std::string bp_string : bp_strings)
```

Comment:
Is the copy here intentional?

Reply:
Yes, it's needed for the StringPopFrontIf function.